pkg/volume/azure_dd/azure_provision.go

```diff
@@ -244,15 +247,16 @@ func (p *azureDiskProvisioner) Provision(selectedNode *v1.Node, allowedTopologie
 		}
 
```

Comment:
could we add a validation of diskEncryptionSetID before other operations? it needs to be in format `"/subscriptions/{subs-id}/resourceGroups/{rg-name}/providers/Microsoft.Compute/diskEncryptionSets/{diskEncryptionSet-name}"`.

Reply:
fixed